exceptions.api.code.4104=预期的错误，手动抛出的错误.